flink-kubernetes/src/test/java/org/apache/flink/kubernetes/KubernetesExtension.java

```diff
@@ -44,14 +48,14 @@ public class KubernetesResource extends ExternalResource {
 
     public static void checkEnv() {
         final String kubeConfigEnv = System.getenv("ITCASE_KUBECONFIG");
-        Assume.assumeTrue(
-                "ITCASE_KUBECONFIG environment is not set.",
-                !StringUtils.isNullOrWhitespaceOnly(kubeConfigEnv));
+        assertThat(StringUtils.isNullOrWhitespaceOnly(kubeConfigEnv))
```

Comment:
Let me confirm that this line of code should reverts to the state before modification, so that the test will be skipped instead of failed in the CI environment without k8s, right?

Reply:
you should use `org.assertj.core.api.Assumptions.assumeThat`